CHANGELOG.md

```diff
@@ -1,11 +1,55 @@
 # Changelog
 
 
+## Version 2.0.0-alpha.19
+
+- User can define keyboard shortcuts for any menu command as well as several
+  other commands that are not in menus. Use Help-->Command Palette, then 
+  Edit Shortcut. 
+- Ebookmaker can now be run via API without installing it on local computer,
+  thus always running latest version installed at PG
+- Scan images can now be rotated in the image viewer, with the rotation saved
+  permanently
```

Comment:
... with the rotation saved permanently per image rotated

It's just a little bit more specific, and there's lots of room left on the line 😁